mantidproject/mantid
# Fix script window having duplicate files after serialisation

**This should only be merged after #16846**

This ticket is part of the work in #15602 .

**To test:**
- Open a few scripts and save a new project.
- Close and reopen mantid
- Observe scripts are loaded again.
- Save project again.
- Close and reopen mantid
- Observe duplicate scripts loaded
- Apply fix.
- Repeat above and observe that each file is only opened once.

Fixes #17060 .

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
